7-AdvancedScenarios/2-call-api-pop/README.md

```diff
@@ -0,0 +1,255 @@
+
+This sample demonstrates the [Proof of Possession](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/access-token-proof-of-possession.md) (PoP) authentication scheme. This authentication scheme cryptographically binds the access tokens to the browser and client application from which they are requested, meaning they cannot be used from a different application or device. The resource that accepts the PoP token (i.e. a web API) needs to be able to decipher the incoming request for the PoP scheme to work properly. This is achieved by extending the [Microsoft.Identity.Web](https://github.com/AzureAD/microsoft-identity-web) package with [SignedHTTPRequest](./API/Microsoft.Identity.Web.Future/SignedHttpRequest) module.
+
+## Scenario
+
+1. The client Angular SPA uses **MSAL Angular** to sign-in and obtain a JWT access token from **Azure AD**.
+1. The client Angular SPA wraps the access token in a secure *envelope* and stamps it with its signature.
+1. The access token then is used as a *PoP* token to authorize the user to call the .NET Core web API protected by **Azure AD**.
+1. The .NET Core web API decrypts the secure envelope to obtain the access token. Once validated, it responds with the protected resource.
+
+![Overview](./ReadmeFiles/topology.png)
+
+## Contents
+
+| File/folder                         | Description                                                |
+|-------------------------------------|------------------------------------------------------------|
+| `SPA/src/app/auth-config.ts`        | Authentication parameters for SPA project reside here.     |
+| `SPA/src/app/app.module.ts`         | MSAL Angular is initialized here.                          |
+| `API/appsettings.json`              | Authentication parameters for API project reside here.     |
+| `API/Startup.cs`                    | Microsoft.Identity.Web is initialized here.                |
+
+## Prerequisites
+
+- An **Azure AD** tenant. For more information see: [How to get an Azure AD tenant](https://docs.microsoft.com/azure/active-directory/develop/quickstart-create-new-tenant)
+- A user account in your **Azure AD** tenant.
+
+## Setup
+
+### Step 1. Clone or download this repository
+
+```console
+    git clone https://github.com/Azure-Samples/ms-identity-javascript-angular-tutorial.git
+```
+
+or download and extract the repository .zip file.
+
+> :warning: To avoid path length limitations on Windows, we recommend cloning into a directory near the root of your drive.
+
+### Step 2. Install .NET Core API dependencies
+
+```console
+    cd ms-identity-javascript-angular-tutorial
+    cd 3-Authorization-II/1-call-api/API/TodoListAPI
```

Comment:
Path should be: cd 7-AdvancedScenarios/2-call-api-pop/API/TodoListAPI

Reply:
oops, done!